frontend/src/views/Dashboard.js

```diff
@@ -26,13 +28,29 @@ function Dashboard() {
       }
     >
       <div className={styles.dashboard}>
+      <span class="material-symbols-outlined">
+        account_circle
+      </span>
         <div className={styles.title}>Welcome</div>
+        <p>to</p>
+        <div className={styles.title}>Phone a Friend!</div>
+        <div>
+          <span class="material-symbols-outlined" >
+            group
+          </span>
```

Comment:
Looks like you may need to add `Link` and we also need to setup the routes somewhere (per this page in stackoverflow https://stackoverflow.com/questions/44387318/linking-button-to-another-page) I don't know how to do that so I'll let @ValentinaBacherer take the lead on setting up routes 😬

Reply:
Yes, I did try adding Link and set up routes in different ways.  The issue I'm having is when I wrap the icon in the Link, the icon disappears from view.  I'm going to try to download the icon image and add it to the code and try linking the image for routing.